### DISCUSSION TITLE
Game Hub

### DISCUSSION THREAD

**Discussion by cococards64** [TurboWarp/packager#456] [Ideas]
Like steam, this will allow to view, play, and  download  games.  You can select the name  and change the pictures of stuff like the buttons but will have a default picture to begin. people can make their own passwords and username. they can also add their gmail for extra protection but  is optional. whenever you export it  it gives a folder with 3 files, {name}.exe, {name}setup.exe, and  {name}manager.exe . manager will be used to add projects or modify things. in it use a {projectname}.exe file  from scratch project packager to use variables to change to a   set number when making the project  in scratch so when they are registered will add achievements or items or avatars, avatar borders, and page themes. when making  the hub you can checkbox custom avatars or ones earned in games. you can  set  a default  profilepicture/avatar for when people havent earned any. you have no borders  or page by default. you  can  checkbox whether items  are  tradeable or  not  in manager.

**Comment by coconutk5335** [comment]
plez add
